spec/unit/dbconsole_spec.rb

```diff
@@ -0,0 +1,22 @@
+RSpec.describe 'dbconsole' do
```

Comment:
I would tend to agree at this stage @blankenshipz. When originally implementing this feature we had added a `./lib/nib/dbconsole.rb` and therefore added a unit test to match. As of this commit, we found that it was really just a `run`... 

Upon further review though we believe we'll need to break this back out into it's own class. The reason being is the shell is doing some very strange things with quoted strings passed to `nib`. Therefore we will need some special handling of `args` for this command. That being the case we'll likely want to have a unit test. 😉